src/components/molecules/navigationMenu.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+	<ul class="menu menu--hidden item-claster" id="root">
+		<back/>
+		<search/>
+		<li v-bind:item="item" v-bind:key="item.id" v-for="item in menuText">
+				<navigationText/>
+		</li>
+		<navigationExpandableItem/>
+	</ul>
+</template>
+
+<script>
+	import back from "./backItem.vue"
+	import search from "./searchItem.vue"
+	import navigationExpandableItem from "./navigationExpandableItem.vue"
+	import navigationText from "../atoms/navigationText.vue"
+	export default {
+		name: "navigationMenu",
+		components: {
+			back,
+			search,
+			navigationExpandableItem,
+			navigationText
+		},
+		data() {
+			return {
+				menuText: [
+					{
+						id: 1,
+						innerText: "home"
+					},
+					{
+						id: 2,
+						innerText: "about"
+					},
+					{
+						id: 3,
+						innerText: "menu"
+					},
+					{
+						id: 4,
+						innerText: "reservations"
```

Comment:
Okay, here's the thing - in this project you'll have to import all your textual info from the JSON file that is provided on github. So from now on, do not write the text with your own hands, import it instead.
Good luck!